### ISSUE TITLE
[Wallet Redesign] Wallet

### ISSUE THREAD

**Issue by gibbsfromncis** [bitshares/bitshares-ui#2672]

**Comment by startailcoon** [comment]
## Overall
This is pretty much what we have today on the "Portfolio" tab. I like that the table is condensed with the operations for each asset in a operation list instead.

## Favorites
I like this idea, and like we discussed last time, we should allow the user to have a "large" amounts of assets as favorites. I'd say that we should allow at least up to at least 10, if not an unlimited amount. Rows should just be dynamic to fit the favorite boxes to the window width. 

## Asset Table
Hiding assets less than a defined amount of an asset is a good idea. We need to make sure that we don't hide assets that have low trading value and doesn't have a price due to this. Today we take the price from exchange based on the "Asset -> Preferred unit of account". This can sometimes give a price of 0 if Asset:<other asset> isn't traded.

A user should also be able to hide defined assets independent of this option. Both with and without.

**Comment by sschiessl-bcp** [comment]
The user transition must be smooth, I would suggest we keep the wording similar. 

Wallet was always the whole shebang, and what you present is the Dashboard.

I personally like the idea of Rossul that we allow "Widgets" on the Dashboard, with the hope that other devs get inspired to contribute fancy / creative widgets

**Comment by sschiessl-bcp** [comment]
No, I misunderstood. You have Wallet and Dashboard. Maybe this could be "Balances" then?
